content/ru/docs/concepts/cluster-administration/logging.md

```diff
@@ -0,0 +1,205 @@
+
+Однако встроенной функциональности контейнерного движка или среды исполнения обычно недостаточно для организации полноценного решения по сбору логов.
+
+Например, может возникнуть необходимость просмотреть логи приложения при аварийном завершении работы Pod'а, его вытеснении (eviction) или "падении" узла.
+
+В кластере у логов должно быть отдельное хранилище и жизненный цикл, не зависящий от узлов, Pod'ов или контейнеров. Эта концепция называется _сбор логов на уровне кластера_.
+
+<!-- body -->
+
+Архитектуры для сбора логов на уровне кластера требуют отдельного бэкенда для их хранения, анализа и выполнения запросов. Kubernetes не имеет собственного решения для хранения такого типа данных. Вместо этого существует множество продуктов для сбора логов, которые прекрасно с ним интегрируются. В последующих разделах описано, как обрабатывать логи и хранить их на узлах.
+
+## Основы сбора логов в Kubernetes
+
+В примере ниже используется спецификация `Pod` с контейнером для отправки текста в стандартный поток вывода раз в секунду.
+
+{{< codenew file="debug/counter-pod.yaml" >}}
+
+Запустить его можно с помощью следующей команды:
+
+```shell
+kubectl apply -f https://k8s.io/examples/debug/counter-pod.yaml
+```
+
+Результат будет таким:
+
+```console
+pod/counter created
+```
+
+Получить логи можно с помощью команды `kubectl logs`, как показано ниже:
+
+```shell
+kubectl logs counter
+```
+
+Результат будет таким:
+
+```console
+0: Mon Jan  1 00:00:00 UTC 2001
+1: Mon Jan  1 00:00:01 UTC 2001
+2: Mon Jan  1 00:00:02 UTC 2001
+...
+```
+
+Команда `kubectl logs --previous` позволяет извлечь логи из предыдущего воплощения контейнера. Если в Pod'е несколько контейнеров, выбрать нужный для извлечения логов можно с помощью флага `-c`:
+
+```console
+kubectl logs counter -c count
+```
+
+Для получения дополнительной информации см. [документацию по `kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs).
+
+## Сбор логов на уровне узла
+
+![Сбор логов на уровне узла](/images/docs/user-guide/logging/logging-node-level.png)
+
+Среда исполнения для контейнера обрабатывает и перенаправляет любой вывод в потоки `stdout` и `stderr` приложения. Docker Engine, например, перенаправляет эти потоки [драйверу журналирования](https://docs.docker.com/engine/admin/logging/overview), который в Kubernetes настроен на запись в файл в формате JSON.
+
+{{< note >}}
+JSON-драйвер Docker для сбора логов рассматривает каждую строку как отдельное сообщение. В данном случае поддержка многострочных сообщений отсутствует. Обработка многострочных сообщений должна выполняться на уровне лог-агента или выше.
+{{< /note >}}
+
+По умолчанию, если контейнер перезапускается, kubelet сохраняет один завершенный контейнер с его логами. Если Pod выселяется с узла, все соответствующие контейнеры также выселяются вместе с их логами.
```

Comment:
"Eviction" still has an old translation here (twice), let's make it consistent.

Reply:
Oh gosh, sorry, somehow I missed it.